src/TestSuite/Fixture/FixtureManager.php

```diff
@@ -352,11 +376,17 @@ public function loadSingle($name, $db = null, $dropTables = true)
                 $db = ConnectionManager::get($fixture->connection());
             }
 
+            $fixture->dropConstraints($db);
```

Comment:
In the case where you are loading a single fixture do you need to do constraints as a separate step?